README.md

```diff
@@ -163,9 +163,9 @@ for more information.
 Events older then a month are archived to S3, you can import these events back
 into your local DB by running the rake tasks in lib/tasks/events.rake, after
 you set up the relavent ENV variables. For example if you want to find all the
-events that are relavant for a particular content item you can run:
+events that are relavant for a particular content_id you can run:
```

Comment:
Thanks - already got that sorted over here: https://github.com/alphagov/publishing-api/pull/735/commits/c8778326865434dfdc08e779966a2c68eb3eb5bb